### DISCUSSION TITLE
Progressive web rust framework

### DISCUSSION THREAD

**Discussion by edezhic** [swc-project/swc#7838] [I'm doing this]
Hey, I'm working on a restful rust framework that allows cross-compiling UI into wasm which runs inside the service worker [github.com/edezhic/pwrs](https://github.com/edezhic/pwrs) to enable client-side  rendering and PWA installability 
I want to set up basic typescript bundling and ->JS translation to be able to use custom TS optionally bundled with wasm bindings along with Rust
As of now it looks like this - https://github.com/edezhic/pwrs/blob/main/build/swc.rs , maybe you have some suggestions for improvements?

**Comment by kdy1** [comment]
Using `swc_bundler` is not a good idea.

**Reply by kdy1** [reply]
I'd recommend implementing a new bundler.
